homeassistant/components/onewire/binary_sensor.py

```diff
@@ -0,0 +1,103 @@
+"""Support for 1-Wire environment binary sensors."""
```

Comment:
This reallyreally should be done in separate PR to ease review of different features.

Reply:
Removed in 45701c486c6a7e04cbe0d681f6b7f3ffded82976